proposals/using-alias-types.md

```diff
@@ -38,20 +38,25 @@ The sections that do need updating are:
 + The namespace_name referenced by a using_namespace_directive is resolved in the same way as the namespace_or_type_name referenced by a using_alias_directive. Thus, using_namespace_directives in the same compilation unit or namespace body do not affect each other and can be written in any order.
 ```
 
+```diff
++ It is illegal for a using alias type to be a nullable reference type.
 
-## Design meeting open questions.
-
-This section needs to be resolved in a design meeting.
+    1. `using X = string?;` is not legal.
+    2. `using X = List<string?>;` is legal.  The alias is to `List<...>` which is itself not a nullable reference type itself, even though it contains one as a type argument.
+```
 
-The intent of this specification is to allow one to write something like:
+# Supporting aliases to types containing pointers.
 
-```
-using MyPointer = My*;
-```
+A new [unsafe context](https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/unsafe-code.md#222-unsafe-contexts) is added through an optional 'unsafe' modifier in the using_alias_directive production:
 
-The spec is currently unclear if this would be ok or not.  Technically, the `using_alias_directive` here is not in an `unsafe` context, so the `My*` could be considered an error.  However, the spirit of this specification is that should be allowed, and only the *usages* of `MyPointer` would themselves have to either be in another `using_alias_directive` or in an `unsafe` context.  Another way this could be formalized is that the `(namespace_name | type)` portion of a `using_alias_directive` would always be an `unsafe` context, but that wouldn't negate the fact that any place that alias was referenced would also need to be an `unsafe` context.
+```diff
+using_alias_directive
++    : 'using' 'unsafe'? identifier '=' (namespace_name | type) ';'
+    ;
 
++ The 'unsafe' modifier specified in the alias declaration causes the entire textual extent of the 'type' portion to become an unsafe context. 
```

Comment:
>modifier [](http://example.com/codeflow?start=15&length=8)

nit: I might avoid the word "modifier" here, since we're not dealing with a modifier list. For comparison, section 22.2 only uses "`unsafe` modifier" for positions that are actually modifiers.

Reply:
done.